imapclient/imapclient.py

```diff
@@ -109,8 +110,6 @@ class IMAPClient(object):
     calls if required.
 
     """
-
-    Error = imaplib.IMAP4.error
```

Comment:
This would be a good compromise indeed.

The "Exceptions" section in the doc will need to be rewritten to describe each exception class created. I will sligthly change the documentation of `IMAPClient`'s `Error`, `AbortError` and `ReadOnlyError` subclasses, as they will be now considered as classes of `imapclient.exceptions` module.
The `IMAPClient` will still have references to thoses three base classes, for backward compatibility.